packages/expo-av/ios/EXAV/Video/EXVideoView.m

```diff
@@ -118,14 +118,22 @@ - (void)_updateForNewPlayer
   }
 }
 
-- (void)_removeData
-{
-  if (_data) {
-    [_data pauseImmediately];
-    [_data setStatusUpdateCallback:nil];
-    [_exAV demoteAudioSessionIfPossible];
-    _data = nil;
-  }
+- (void)_removeData {
+  EX_WEAKIFY(self);
+  void (^block)(void) = ^{
+    EX_ENSURE_STRONGIFY(self);
+    if (self->_data) {
+      [self->_data cleanup];
+      [self->_data pauseImmediately];
+      [self->_data setStatusUpdateCallback:nil];
+      [self->_exAV demoteAudioSessionIfPossible];
+      self->_data = nil;
```

Comment:
can `[self->_data cleanup]` be safely called multiple times?
I'm wondering because it is called manually here, then the `_data` is set to `nil` which can possibly `dealloc` and thus call `cleanup` again.